qa/tasks/notification_tests.py

```diff
@@ -12,47 +12,28 @@
 
 from teuthology import misc as teuthology
 from teuthology import contextutil
-from teuthology.config import config as teuth_config
 from teuthology.orchestra import run
 
 log = logging.getLogger(__name__)
 
+
 @contextlib.contextmanager
 def download(ctx, config):
-    """
-    Download the bucket notification tests from the git builder.
-    Remove downloaded test file upon exit.
-    The context passed in should be identical to the context
-    passed in to the main task.
-    """
     assert isinstance(config, dict)
-    log.info('Downloading bucket-notification-tests...')
+    log.info('Downloading bucket-notifications-tests...')
     testdir = teuthology.get_testdir(ctx)
+    branch = ctx.config.get('suite_branch', None)
+    repo = ctx.config.get('suite_repo', None)
+    if not branch:
+        raise ValueError("You need to add suite_branch parameter to your command.")
+    if not repo:
+        raise ValueError("You need to add suite_repo parameter to your command.")
+    log.info('Using branch %s for bucket notifications tests', branch)
     for (client, client_config) in config.items():
-        bntests_branch = client_config.get('force-branch', None)
-        if not bntests_branch:
-            raise ValueError(
-                "Could not determine what branch to use for bn-tests. Please add 'force-branch: {bn-tests branch name}' to the .yaml config for this bucket notifications tests task.")
-
-        log.info("Using branch '%s' for bucket notifications tests", bntests_branch)
-        sha1 = client_config.get('sha1')
-        git_remote = client_config.get('git_remote', teuth_config.ceph_git_base_url)
         ctx.cluster.only(client).run(
-            args=[
-                'git', 'clone',
-                '-b', bntests_branch,
-                git_remote + 'ceph.git',
-                '{tdir}/ceph'.format(tdir=testdir),
-                ],
+            args=['git', 'clone', '-b', branch, repo + '.git', '{tdir}/ceph'.format(tdir=testdir)],
             )
-        if sha1 is not None:
```

Comment:
you can get sha1 from the commandline as well

Reply:
Okay, so I'll keep this part.